rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -2949,7 +3031,32 @@ error_code sys_fs_get_mount_info(ppu_thread&, vm::ptr<CellFsMountInfo> info, u32
 
 error_code sys_fs_mount(ppu_thread&, vm::cptr<char> dev_name, vm::cptr<char> file_system, vm::cptr<char> path, s32 unk1, s32 prot, s32 unk3, vm::cptr<char> str1, u32 str_len)
 {
-	sys_fs.todo("sys_fs_mount(dev_name=%s, file_system=%s, path=%s, unk1=0x%x, prot=0x%x, unk3=0x%x, str1=%s, str_len=%d)", dev_name, file_system, path, unk1, prot, unk3, str1, str_len);
+	sys_fs.warning("sys_fs_mount(dev_name=%s, file_system=%s, path=%s, unk1=0x%x, prot=0x%x, unk3=0x%x, str1=%s, str_len=%d)", dev_name, file_system, path, unk1, prot, unk3, str1, str_len);
+
+	const auto [path_error, vpath] = translate_to_sv(path);
+
+	if (path_error)
+	{
+		return { path_error, vpath };
+	}
+
+	vfs::mount(vpath, lv2_fs_object::get_vfs(vpath));
+
+	return CELL_OK;
+}
+
+error_code sys_fs_unmount(ppu_thread&, vm::cptr<char> path, s32 unk1, s32 unk2)
+{
+	sys_fs.warning("sys_fs_unmount(path=%s, unk1=0x%x, unk2=0x%x)", path, unk1, unk2);
+
+	const auto [path_error, vpath] = translate_to_sv(path);
+
+	if (path_error)
+	{
+		return { path_error, vpath };
+	}
+
+	vfs::unmount(vpath);
```

Comment:
Thanks, but I would like that this line be commented as of now.

Reply:
It's because it's unsafe to put it in the open pike that.

Reply:
I'll also add a root permission check for this syscall.
Just wonder when we should uncomment `vfs::unmount(vpath);`.